linux/keyman-config/tests/test_ibus_util.py

```diff
@@ -30,74 +31,62 @@ def test_getIbusBus_GlobalEngineNotEnabled_ReturnsNone(self, MockIbusBusClass):
         self.assertTrue(MockIbusBusClass.called, "IBus.Bus called")
         self.assertTrue(mock_ibusBusInstance.destroy.called)
 
-    def test_getIbusBus_ReturnsBus(self, MockIbusBusClass):
+    def test_getIbusBus_ReturnsBus(self, patched_time_sleep, MockIbusBusClass):
         # Setup
         mock_ibusBusInstance = MockIbusBusClass.return_value
         # Execute/Verify
         self.assertIsNotNone(get_ibus_bus())
         self.assertTrue(MockIbusBusClass.called, "IBus.Bus called")
         self.assertFalse(mock_ibusBusInstance.destroy.called)
 
-    @patch('keyman_config.gsettings.GSettings.get')
+    @patch('keyman_config.gsettings.GSettings.get', return_value=['k1', 'k2', 'k3'])
     @patch('keyman_config.gsettings.GSettings.set')
-    def test_installToIbus_AlreadyInstalled(self, MockSettingsSet, MockSettingsGet, MockIbusBusClass):
+    def test_installToIbus_AlreadyInstalled(self, patched_settings_set, patched_settings_get, patched_time_sleep, MockIbusBusClass):
         # Setup
-        mock_settingsGet = MockSettingsGet
-        mock_settingsSet = MockSettingsSet
-        mock_settingsGet.return_value = ['k1', 'k2', 'k3']
         mock_ibusBusInstance = MockIbusBusClass.return_value
         # Execute
         install_to_ibus(mock_ibusBusInstance, 'k3')
         # Verify
-        mock_settingsSet.assert_called_once_with(
+        patched_settings_set.assert_called_once_with(
             "preload-engines", ['k1', 'k2', 'k3'], 'as')
         mock_ibusBusInstance.preload_engines.assert_called_once_with(
             ['k1', 'k2', 'k3'])
 
-    @patch('keyman_config.gsettings.GSettings.get')
+    @patch('keyman_config.gsettings.GSettings.get', return_value=['k1', 'k2', 'k3'])
     @patch('keyman_config.gsettings.GSettings.set')
-    def test_installToIbus_InstallsNewKb(self, MockSettingsSet, MockSettingsGet, MockIbusBusClass):
+    def test_installToIbus_InstallsNewKb(self, mock_settings_set, mock_settings_get, patched_time_sleep, MockIbusBusClass):
```

Comment:
You used `patched_settings_set` above in line 44, but `mock_settings_set` here. Is that intentional?

Reply:
Good point. Done.